nemo/utils/export_utils.py

```diff
@@ -186,12 +186,10 @@ def verify_runtime(model, output, input_examples, input_names, check_tolerance=0
         logging.warning(f"ONNX generated at {output}, not verified - please install onnxruntime_gpu package.\n")
         onnx.checker.check_model(onnx_model, full_check=True)
         return
-
+    del onnx_model
     onnx_session_opt = onnxruntime.SessionOptions()
     onnx_session_opt.graph_optimization_level = onnxruntime.GraphOptimizationLevel.ORT_ENABLE_ALL
-    sess = onnxruntime.InferenceSession(
-        onnx_model.SerializeToString(), sess_options=onnx_session_opt, providers=['CUDAExecutionProvider']
-    )
+    sess = onnxruntime.InferenceSession(output, sess_options=onnx_session_opt, providers=['CUDAExecutionProvider'])
```

Comment:
Hmm this works inside the export engine, but what about when you want to restore a onnx model ? You'll still need to pass SerializeToString() no ?

Reply:
no, you cam always use a file. SerializeToString() is just not compatible with external weights when ONNX size is > 2GB.

Reply:
Oh interesting. I need to update the RNNT script to support this type of loading